libs/gpu/src/main.zig

```diff
@@ -28,7 +34,88 @@ pub const TextureView = @import("texture_view.zig").TextureView;
 
 pub const dawn = @import("dawn.zig");
 
-pub usingnamespace @import("types.zig");
+pub const array_layer_count_undef = @import("types.zig").array_layer_count_undef;
```

Comment:
I mentioned this earlier in Discord:

> my suggestion for callbacks.zig and types.zig is to move all that code directly into main.zig

I think that:

1. The fact that we need to re-export so many declarations, that you felt the need to write a script to do so, is a good hint that something is wrong :)
2. Elevating those declarations right into main.zig eliminates that problem (and the problem of keeping this list of re-exported declarations up to date in the future)
3. Having two files named `types.zig` and `callbacks.zig` is a bit of an anti-pattern in general, anyway - so best we move away from that.

Does that sound reasonable?

Reply:
Done

Reply:
Done